dask/base.py

```diff
@@ -416,14 +416,32 @@ def tokenize(*args, **kwargs):
     return md5(str(tuple(map(normalize_token, args))).encode()).hexdigest()
 
 
+def dont_optimize(dsk, keys):
+    return dsk
+
+
+def optimization_function(obj):
+    if isinstance(obj, type):
+        cls = obj
+    else:
+        cls = type(obj)
+    name = cls.__name__.lower() + '_optimize' # dask.set_options(array_optimize=foo)
+    if name in _globals:
+        return _globals[name] or dont_optimize
```

Comment:
Is the intent here to support `dask.set_options(array_optimize=None)`?

If so, great, but let's document it, test it and explicitly check against `None` rather than implicit falseness. It's not likely, but I can imagine someone writing a callable class that also implements `__bool__`.